calc_controller.py

```diff
@@ -0,0 +1,48 @@
+
+from flask import Flask, redirect, url_for, request
+import math
+app = Flask(__name__)
+
+
+@app.route('/result/<a>/<b>/<value>/<operation>')
+def result(a, b, value, operation):
+
+    if operation == 'add':
+        return f'{a} + {b} = {value}'
+    elif operation == 'subtract':
+        return f'{a} - {b} = {value}'
+    elif operation == 'multiply':
+        return f'{a} * {b} = {value}'
+    elif operation == 'divide':
+        return f'{a} / {b} = {value}'
+    elif operation == 'log':
+        return f'log {a}  = {value}'
+    elif operation == 'log10':
+        return f'log10 {a}  = {value}'
```

Comment:
Also, there is repetition of the 'if operation == ' & 'elif operation == ' conditions between this function and the calc function. Is there a way to avoid it?